src/GovUk.Education.ExploreEducationStatistics.Content.Model/ContentSection.cs

```diff
@@ -133,7 +135,7 @@ public DataBlock()
         
         public string Source { get; set; }
 
-        public DataBlockRequest DataBlockRequest { get; set; }
+        public ObservationQueryContext DataBlockRequest { get; set; }
```

Comment:
It looks like we're replacing usage of DataBlockRequest almost everywhere, including the entity model.  Given that, can we remove it and rename ObservationQueryContext?

Reply:
`DataBlockRequest` has already been removed.
It hasn't been renamed (yet) because it was primarily a Data API request class and Data API doesn't know anything about the concept of data blocks. There is also a `SubjectMetaQueryContext` (similar but has no filters) so I was keeping the name consistent with that.